source/compiler/aslcompiler.l

```diff
@@ -156,7 +156,7 @@
 
 #include <stdlib.h>
 #include <string.h>
-YYSTYPE AslCompilerlval;
+#define YYSTYPE AslCompilerlval;
```

Comment:
I'm taking a look at this right now but I don't have access to gcc-10 (yet). What happens when you remove this #define? Could you show me the error that you get?

Reply:
Indeed, this line can be removed without error. Want me to remove it?